aplus/settings.py

```diff
@@ -182,6 +182,11 @@
 SOCIAL_AUTH_URL_NAMESPACE = 'social'
 SOCIAL_AUTH_USERNAME_IS_FULL_EMAIL = True
 
+##########################################################################¨
+# Student Information System plugin settings
+#SIS_PLUGIN_MODULE = 'course.sis_test'
```

Comment:
I left this unchanged for now. get_sis_configuration() should be the only place using this, and on the other hand, I think it is safer to check hasattr() there in any case. I don't think we should use course.sis_test by default, because some installations do not want to have SIS set up, which is the old behavior and therefore good default.